src/components/breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="flex gap-2">
+    <VaIconMenuCollapsed
+      :class="{ 'x-flip': isSidebarMinimized }"
+      :color="collapseIconColor"
```

Comment:
![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/dac3639e-f9cf-4fb7-8f34-0e36d6714439)

This mode has to go. We won't use it.

Sidebar would be either fully closed or fully expanded.